docs/guides/mongodb/schema-manager/initializing-with-snapshot/index.md

```diff
@@ -0,0 +1,376 @@
+- `spec.terminationPolicy` specifies what KubeDB should do when a user try to delete the operation of MongoDB CR. *Wipeout* means that the database will be deleted without restrictions. It can also be "Halt", "Delete" and "DoNotTerminate". Learn More about these [HERE](https://kubedb.com/docs/latest/guides/mongodb/concepts/mongodb/#specterminationpolicy).
+
+Let’s save this yaml configuration into `mongodb.yaml` Then create the above `MongoDB` CR
+
+```bash
+$ kc apply -f mongodb.yaml 
+mongodb.kubedb.com/mongodb created
+```
+
+### Deploy Vault Server
+
+In this section, we are going to deploy a Vault Server. Let’s deploy it using this following yaml,
+
+```yaml
+apiVersion: kubevault.com/v1alpha1
+kind: VaultServer
+metadata:
+  name: vault
+  namespace: demo
+spec:
+  version: 1.8.2
+  replicas: 3
+  allowedSecretEngines:
+    namespaces:
+      from: All
+    secretEngines:
+      - mongodb
+  unsealer:
+    secretShares: 5
+    secretThreshold: 3
+    mode:
+      kubernetesSecret:
+        secretName: vault-keys
+  backend:
+    raft:
+      path: "/vault/data"
+      storage:
+        storageClassName: "standard"
+        resources:
+          requests:
+            storage: 1Gi
+  authMethods:
+    - type: kubernetes
+      path: kubernetes
+  terminationPolicy: WipeOut
+```
+
+Here,
+
+- `spec.version` is a required field that specifies the original version of Vault that has been used to build the docker image specified in `spec.vault.image` field.
+- `spec.replicas` specifies the number of Vault nodes to deploy. It has to be a positive number.
+- `spec.allowedSecretEngines` defines the types of Secret Engines & the Allowed namespaces from where a `SecretEngine` can be attached to the `VaultServer`.
+- `spec.unsealer` is an optional field that specifies `Unsealer` configuration. `Unsealer` handles automatic initializing and unsealing of Vault.
+- `spec.backend` is a required field that specifies the Vault backend storage configuration. KubeVault operator generates storage configuration according to this `spec.backend`.
+- `spec.authMethods` is an optional field that specifies the list of auth methods to enable in Vault.
+- `spec.terminationPolicy` is an optional field that gives flexibility whether to nullify(reject) the delete operation of VaultServer crd or which resources KubeVault operator should keep or delete when you delete VaultServer crd. 
+
+Let’s save this yaml configuration into `vault.yaml` Then create the above `VaultServer` CR
+
+```bash
+$ kubectl apply -f vault.yaml
+vaultserver.kubevault.com/vault created
+```
+
+
+### Create Repository Secret
+
+Here, we are using local backend for storing data snapshots. It can be a cloud storage like GCS bucket, AWS S3, Azure Blob Storage etc. or a Kubernetes persistent volume like HostPath, PersistentVolumeClaim, NFS etc. For more information check [HERE](https://stash.run/docs/latest/guides/backends/overview/)
```

Comment:
`or a Kubernetes persistent volume like HostPath, PersistentVolumeClaim, NFS etc` .  This sentence doesn't make any sense.  You mean `kubernetes native resources like ...` ?